parsers/rust.c

```diff
@@ -97,8 +97,7 @@ static void parseBlock (lexerState *lexer, bool delim, int kind, vString *scope)
 /* Resets the scope string to the old length */
 static void resetScope (vString *scope, size_t old_len)
 {
-	scope->length = old_len;
-	scope->buffer[old_len] = '\0';
+	vStringTruncate (scope, old_len);
```

Comment:
The function is only used once, and this body is now only 1 call with the same args (e.g. basically an alias for `vStringTruncate`), it could probably be simply removed.  But if you wanna keep the naming for clarity, it also sounds reasonable.

Reply:
I would like to keep this because it may help code reader understand the intent of code.